lib/setup.js

```diff
@@ -1,22 +1,67 @@
 const path = require('path');
+const fs = require('fs');
+const vite = require('./vite.js');
+const getPort = require('get-port')
 
 const vite_dev_server = {
-  port: 5174
+  port: process.argv.includes('--test-worker') ? null : 5173
 };
 try {
   const nightwatch_config = require(path.resolve('./nightwatch.conf.js'));
   Object.assign(vite_dev_server, nightwatch_config.vite_dev_server || {});
 } catch (err) {}
 
-const vite = require('./vite.js');
+const projectJsConfigFile = path.join(process.cwd(), 'vite.config.js');
+const projectTsConfigFile = path.join(process.cwd(), 'vite.config.ts');
+
+const hasProjectJsConfigFile = () => {
+  try {
+    if(fs.statSync(projectJsConfigFile).isFile()) {
+      return projectJsConfigFile;
+    }
+  } catch(err) {
+    if(fs.statSync(projectTsConfigFile).isFile()) {
+      return projectTsConfigFile;
+    }
+
+    return false;
```

Comment:
nit pick: I think we should send `''` instead of `false`

Reply:
how does it matter?

Reply:
It doesn't, but ideally, it can introduce bugs in future, as we are sending both `boolean | string`.